test/functional/test_files.py

```diff
@@ -103,6 +107,11 @@
 ]
 
 
+def calc_path_rel_to_cwd(p):
+	prefix = os.path.commonprefix([p, os.getcwd()])
+	relpath = os.path.relpath(p, prefix)
```

Comment:
Use `str(p)` instead of `p` here to make the Python 3.5- error go away.

Reply:
Fixed.